modules/mediawiki/files/bin/deploy-mediawiki.py

```diff
@@ -33,12 +33,34 @@ def run(args, start):
         os.system(f'/usr/local/bin/logsalmsg {text}')
     else:
         print(text)
+    if args.world:
+        if args.pull:
+            pull = str(args.pull).split(',')
+            if 'world' not in pull:
+                pull.append('world')
+                args.pull = ','.join(pull)
+            pull = None
```

Comment:
Why reset pull here

Reply:
I guess that isn't necessary yeah, it was mainly because this instance of pull is being conditionally set if world is used, whereas its set again under args.pull directly.

Reply:
see code suggestion